server.js

```diff
@@ -0,0 +1,26 @@
+import http from 'http';
+import url from 'url';
+import fs from 'fs';
+
+const PORT = 8080;
+const index = fs.readFileSync('index.html');
+const events = fs.readFileSync('./dist/events.json');
+
+function createServer(req, res) {
```

Comment:
Change name to `initServer` so it’s less confusing and won’t conflict with the changed http import.

Reply:
Right